spooky-doodle/Trestlebridge-Farms
# Total plants

# Description
#15 
Given a user has launched the program
When the user selects the option to purchase seed
Then the user should be presented with a list of seeds to purchase

Given user is viewing a list of seeds
When the user selects a seed
Then the user should be presented with a list of fields that are appropriate for that seed
And the current total number of plants currently in each facility should be displayed

----------------------------------------------------
#16 

Given a user has launched the program
When the user selects the option to purchase seed
Then the user should be presented with a list of seeds to purchase

Given user is viewing a list of seeds
When the user selects a seed
Then the user should be presented with a list of fields that are appropriate for that seed AND have enough room for the new seed

Given user is viewing the list of appropriate potential seed fields
When the user selects a field that has available capacity
Then the user should be presented with a message confirming the field selection
And the user should be taken back to the main menu
----------------------------------------------------
#17

Given a user has launched the program
When the user selects the option to purchase seed
Then the user should be presented with a list of seeds to purchase

Given user is viewing a list of seeds
When the user selects a seed
Then the user should be presented with a list of potential seed fields
And the current total number of plants currently in each field should be displayed
And the current count of each type of plant currently in each field should be displayed


Fixes # (15, 16, 17)

## Type of change

Please delete options that are not relevant.


# Testing Instructions

dotnet run to start application
add multiple plowed fields and natural fields
place each type of seed in the corresponding fields
ensure that the field displays update with the appropriate amount of seeds and space in each field.
ensure a full field is not displayed


# Checklist:
